port/unix/omrsysinfo.c

```diff
@@ -3517,25 +3518,46 @@ readCgroupFile(struct OMRPortLibrary *portLibrary, int pid, BOOLEAN inContainer,
 	}
 
 	while (0 == feof(cgroupFile)) {
+		char buffer[PATH_MAX + 1024];
```

Comment:
The PATH_MAX variable implies that is the maximum buffer size.  How come you are adding 1024 to this size?

Reply:
This is somewhat arbitrary - previously group and subsystems were of max size PATH_MAX and 1024 respectively, and since buffer is to contain both of them, this is the size I chose. Realistically, no entry will get close to 1024, and PATH_MAX itself is a macro for 1024. Should I just change all of these to be of length PATH_MAX?

Reply:
Done. Also removed now unnecessary overflow checks (as fgets is capped at the number of characters it will read)